python/oak/mapping_visu.py

```diff
@@ -75,7 +75,6 @@ def __init__(self, voxelSize, cameraFollow, colorOnly):
 
     def run(self):
         self.vis.create_window()
-        self.vis.set_full_screen(True)
```

Comment:
You could also make this a command line argument.

Reply:
The Open3D toggles between fullscreen when the user presses 'Alt + Enter', but I guess an option to start in fullscreen could still be useful.

Reply:
OK. If it's already included so easly, then no need to add